testuseradd.py

```diff
@@ -0,0 +1,28 @@
+"""
+This test checks addition of user 
+"""
+import unittest
+from .testutils import system
+
+
+class UserAddtest(unittest.TestCase):
+    """
+    Useradd test
+    """
+
+    def test_useradd(self):
+        out, err, retcode = system("sudo useradd test_satyajit")
+        self.assertEqual(retcode, 0, out)
+        out, err, retcode = system("cat /etc/passwd")
+        self.assertEqual(retcode, 0, out)
```

Comment:
We can skip checking `retcode` here since our purpose is just to check the user that you have already done on line no. #19.